pkg/cmd/admin/network/isolate_projects.go

```diff
@@ -0,0 +1,78 @@
+package network
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	kerrors "k8s.io/kubernetes/pkg/util/errors"
+
+	"github.com/openshift/openshift-sdn/plugins/osdn/multitenant"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+)
+
+const (
+	IsolateProjectsNetworkCommandName = "isolate-projects"
+
+	isolateProjectsNetworkLong = `
+Isolate project network
+
+Allows projects to isolate their network from other projects in case of %[1]s network plugin.`
```

Comment:
I think "Allows projects to isolate their network from other projects (when using the %[1]s network plugin)." would be clearer.